### ISSUE TITLE
Entire page is blacked out when another add-on overlays content area

### ISSUE THREAD

**Issue by blackwind** [m-khvoinitsky/dark-background-light-text-extension#25]
When an add-on overlays the page, DB&LT applies its filters to the add-on as well, resulting in an entirely blacked-out page. In particular, this is a huge problem for:

* [Snap Links Plus](https://addons.mozilla.org/en-US/firefox/addon/snaplinksplus/) (v3 and above) - Hold the right mouse button and drag to select links. Entire page turns black, so you can't see what you're selecting.

* [uBlock Origin](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/) - Click the toolbar button, then the eyedropper icon to enter element picker mode. Entire page turns black, so you can't see what you're selecting.

I'm not sure there's a good solution to this short of manually blacklisting the element ids for each add-on.

**Comment by blackwind** [comment]
Still an issue with uBlock Origin in v0.5.6.

**Comment by blackwind** [comment]
Still an issue in v0.5.7.

**Comment by m-khvoinitsky** [comment]
> Still an issue with uBlock Origin in v0.5.6.

Indeed. There was a small mistake in previous fix. Fixed it in 0de708068a4555bf2b99adcdc60a806b3f0272ec
Will upload a fixed version to AMO in a few minutes

**Comment by blackwind** [comment]
Seems to be fixed now except on pages using Simple CSS mode. Is it possible to fix that as well?

**Comment by m-khvoinitsky** [comment]
@blackwind , I apologize for so late reply.
Unfortunatey, no way for Simple CSS - it's impossible.
It's a kind of problem why Stylesheet processor was created.
